packages/plugins/plugin-idfa/src/IdfaPlugin.tsx

```diff
@@ -3,36 +3,64 @@ import {
   PluginType,
   SegmentClient,
 } from '@segment/analytics-react-native';
-import { IdfaEvents } from './IdfaEvents';
 import type { IdfaData } from './types';
 import { AnalyticsReactNativePluginIdfa } from './AnalyticsReactNativePluginIdfa';
 
 const { getTrackingAuthorizationStatus } = AnalyticsReactNativePluginIdfa;
 
 export class IdfaPlugin extends Plugin {
   type = PluginType.enrichment;
+  private isEnabled?: boolean = false;
+  private isDisabled?: boolean;
+
+  constructor(enabled?: boolean) {
```

Comment:
Add some JSDoc comments as this constructor receives an additional parameter that should be documented.
I might also suggest renaming from `enabled` to `shouldAskPermissions` or something similar as I explained above.

Reply:
Also the constructor in my opinion should default to `true`, which is kinda like the behavior it already has but more explicit.